public/components/data_connections/components/data_connection.tsx

```diff
@@ -37,64 +40,68 @@ export const DataConnection = (props: any) => {
     allowedRoles: [],
     name: '',
     cluster: '',
+    connector: '',
+    properties: {},
   });
   const [hasAccess, setHasAccess] = useState(true);
-  const { http } = coreRefs;
+  const { http, chrome } = coreRefs;
 
   useEffect(() => {
-    http
+    chrome!.setBreadcrumbs([
+      {
+        text: 'Data Connections',
+        href: '#/',
+      },
+      {
+        text: `${dataSource}`,
+        href: `#/manage/${dataSource}`,
+      },
+    ]);
+    http!
       .get(`${DATACONNECTIONS_BASE}/${dataSource}`)
       .then((data) =>
         setDatasourceDetails({
           allowedRoles: data.allowedRoles,
           name: data.name,
           cluster: data.properties['emr.cluster'],
+          connector: data.connector,
+          properties: data.properties,
         })
       )
       .catch((err) => {
         if (err.body.statusCode === 403) {
```

Comment:
Without re-throwing, you're effectively swallowing all other errors except 403.  Is that really what we want?
If any/ever error should be met with setHasAccess(false), then just set it without condition.
This might be a case to wrap this in a function so you can tell a story `checkStatusForbiddenOrThrow(err)`.